rest/model/notification.go

```diff
@@ -104,37 +84,6 @@ type APISlack struct {
 	Attachments []APISlackAttachment `json:"attachments"`
 }
 
-// BuildFromService converts from service level message.Slack to APISlack.
-func (n *APISlack) BuildFromService(h interface{}) error {
```

Comment:
Removed this function because it actually doesn't work (at 123 we pass in a slack.Attachment but we error if it's not a message.SlackAttachment?) and we aren't using it anywhere.